### ISSUE TITLE
Xbox Game Bar

### ISSUE THREAD

**Issue by xx1182** [ClassicOldSong/Apollo#156]
Is It possible? I've heard it uses some weird ways that connects with direct3d so won't work with.

Regards.

**Comment by ClassicOldSong** [comment]
It works.

**Comment by xx1182** [comment]
> It works. Maybe you're asking about something more particular, in that case would you modify the title and description into a more detailed one?

I'm referring to "Windows + G"... Windows Gamebar ..

Regards.

**Comment by ClassicOldSong** [comment]
Hmmm I thought it worked at least once, but just tested it's not working with default capture method and WGC is not usable right now. So it won't show on the stream but will actually presents on the screen.

**Comment by monogoat** [comment]
> Hmmm I thought it worked at least once, but just tested it's not working with default capture method and WGC is not usable right now. So it won't show on the stream but will actually presents on the screen.

I would also love the game bar to be usable via stream, I am currently running Sunshine in WGC mode but can't run it as a service this way so I have an annoying terminal open at all times.

**Comment by ClassicOldSong** [comment]
Not running as a service/at system privilege will lead to other problems: UAC dialogs will not be accessable, login password UI cannot input and cannot be captured.

**Comment by monogoat** [comment]
> Not running as a service/at system privilege will lead to other problems: UAC dialogs will not be accessable, login password UI cannot input and cannot be captured.

Yep, was just mentioning I'm stuck with Sunshine and those issues at the moment to be able to render the gamebar. Thank you for your hard work!

**Comment by xx1182** [comment]
> Not running as a service/at system privilege will lead to other problems: UAC dialogs will not be accessable, login password UI cannot input and cannot be captured.

The graphics card overlay (AMD adrenaline) doesn't work.
SteamOverlay works... 

Maybe setting the second display as a first display(numero 1) would resolve the issiue?

**Comment by ClassicOldSong** [comment]
Those overlay usually draw on the pipeline after the image is captured. Using a capture card is the ultimate solution to this problem but it introduces more latency.

**Comment by xx1182** [comment]
> Those overlay usually draw on the pipeline after the image is captured. Using a capture card is the ultimate solution to this problem but it introduces more latency.

I set the display mode TO: **EXTEND** and the "AMD OVERLAY" is available

Additionally, when the gesture "hold 4 fingers" is pressed, the gamebar shows up.

![Image](https://github.com/user-attachments/assets/16bcc5ce-3588-45df-9343-75c2cdba9c64)

**Comment by xx1182** [comment]
No

**Comment by ClassicOldSong** [comment]
?

**Comment by xx1182** [comment]
> ?

I pressed "closed the issue" and I was replaying to myself that it wasn't, sorry.

Btw there is a problem, when setting the display type as extended, when launching an app it runs on display 1. It's easily resolvable by moving the window to the stream, but not optimal

**Comment by ClassicOldSong** [comment]
Unless the app has an option for that, they'll always start on the primary display. In Extended mode you can set the virtual display as primary.
